climpred/tests/conftest.py

```diff
@@ -280,3 +280,30 @@ def multi_dim_ds():
     ds = xr.tutorial.open_dataset('air_temperature')
     ds = ds.assign(**{'airx2': ds['air'] * 2})
     return ds
+
+
+@pytest.fixture
+def da_SLM():
+    """Small xr.DataArray with dims `S`, `M` and  `L` for `init`, `member` and
+    `lead`."""
+    lead = np.arange(5)
+    init = np.arange(5)
+    member = np.arange(5)
+    return xr.DataArray(
+        np.random.rand(len(lead), len(init), len(member)),
+        dims=['S', 'L', 'M'],
+        coords=[init, lead, member],
+    )
+
+
+@pytest.fixture
+def da_dcpp():
+    """Small xr.DataArray with coords `init` and `lead`."""
```

Comment:
This description seems wrong. Should be with dims `dcpp_init_year`, `time`, and `member_id`